plugins/syslog.yaml

```diff
@@ -36,13 +36,17 @@ pipeline:
   - id: syslog_input
     type: udp_input
     listen_address: {{ $listen_address }}
+    labels:
+      log_type: syslog
     output: syslog_parser
 {{ end }}
 
 {{ if eq $connection_type "tcp" }}
   - id: syslog_input
     type: tcp_input
     listen_address: {{ $listen_address }}
+    labels:
+      log_type: syslog
     output: syslog_parser
 {{ end }}
```

Comment:
This one seems to be missing the `syslog_parser`

Reply:
I think you just missed it down below. It exists outside of the conditional statements. That way we didn't need to define the same parser twice for each conditional.